lizrd/scripts/grid_utils.py

```diff
@@ -27,6 +27,17 @@ def get_machine_backend() -> MachineBackend:
         return MachineBackend.LOCAL
 
 
+def get_cache_path(runner: MachineBackend) -> str:
+    if runner == MachineBackend.ATHENA:
+        return "/net/tscratch/people/plgjkrajewski/.cache"
+    elif runner == MachineBackend.IDEAS:
+        return "/raid/NFS_SHARE/llm-random/.cache"
+    elif runner == MachineBackend.ENTROPY_GPU:
+        return "/home/jkrajewski/.cache"
+    else:
+        return "/tmp/.cache"
```

Comment:
Doesn't this lead to cache disappearing on every reboot and datasets having to be downloaded again?